docs/function-controller-configuration.md

```diff
@@ -0,0 +1,68 @@
+# Controller configurations for Functions
+### Using `ConfigMap`
+Configurations for functions can be done in `ConfigMap`: `kubeless-config` which is a part of `Kubeless` deployment manifests.
+
+Deployments for function can be configured in `data` inside the `ConfigMap`, using key `deployment`, which takes a string in the form of `yaml/json` and is driven by the structure of [v1beta2.Deployment](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#deployment-v1beta2-apps):
+E.g.
+```yaml
+---
+apiVersion: v1
+data:
+  deployment: |-
+    {
+      "metadata": {
+          "annotation":
+            "annotation-to-deployment": "value"
+      },
+      "spec": {
+        "replicas": 1
+        "template":
+          "annotations":
+            "annotations-to-pod": "value"
+      }
+    }
+  ingress-enabled: "false"
+  service-type: ClusterIP
+kind: ConfigMap
+metadata:
+  name: kubeless-config
+  namespace: kubeless
+```
+
+This in turn will add the config to all the function `deployments` created by controller.
```

Comment:
Should we be clear on which configurations should be added at this place ? Or giving some recommendations or examples would be helpful

Reply:
Fixed docs